### PR TITLE
SCT-515: Update types

### PR DESCRIPTION
**Pull request by JamesJeffryes**
This is my first crack at types for Media & FBAModel types as well as updates to match the new genome object

### CODE REVIEW

File: resources/typemappings/Genome.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    KBaseGenome.Genome:
+        default:
+           -
+               type: Genome
+               version: 1
+           -
+               type: GenomeFeature
+               version: 1
+
+        versions:
```
Comment (MrCreosote) [inline resources/typemappings/Genome.yaml]: Don't mix production and CI specs in the same mapping - there will have to be one mapping for production and one for CI. This mapping will map ci version 9 to the search version 2

File: resources/types/FBAModel.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            {
+                "source-key": "genome_guid",
+                "transform": "lookup.key.taxonomy",
+                "full-text": true,
+                "key-name": "taxonomy",
+                "ui-name": "Genome Taxonomy"
+            },
+            {
+                "source-key": "genome_guid",
+                "transform": "lookup.key.id",
```
Comment (MrCreosote) [inline resources/types/FBAModel.json]: Is the genome id something that could change?

**Comment by JamesJeffryes** [comment]
I suppose tests should be written for the new parsed types

**Comment by MrCreosote** [comment]
I would just test the types manually, personally. At some point we'll hopefully write a validator for the specs.

**Comment by arfathpasha** [comment]
Looks like you may have used 'git fetch' or 'git rebase' as I see 37 files changed. This makes it hard to review as the PR includes changes from other PRs. I have made this mistake previously and the way around it, I found, is to stick to 'git pull' from upstream. The pull command merges all upstream changes as a single commit into your origin and the PRs come out with only the changes/commits that you introduced into your origin.

**Comment by JamesJeffryes** [comment]
Sorry about that Arfath. Is there any good way to clean up the PR now?

**Comment by arfathpasha** [comment]
I suggest closing this PR and creating a new one. Safest, easiest, fastest way is to 
- save the files you changed. 
- delete your origin repo
- fork again from upstream
- put back the files you created/modified into origin
- create PR

Else you'd have to cherry-pit (yucky opposite of cherry picking)
https://sethrobertson.github.io/GitFixUm/fixup.html

**Pull request closed without merging**